### PR TITLE
break it again

### PR DESCRIPTION
**Pull request by leixu216**

### CODE REVIEW

**Comment by leixu216** [comment]
@ups216 could you please take a look.

File: api/controllers/arithmeticController.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ exports.calculate = function(req, res) {
 
   var operations = {
     // +a will convert a into integer type to make sure the operation is correct
-    'add':      function(a,b) { return +a + +b },
+    'add':      function(a,b) { return a + b },
```
Comment (ups216) [inline api/controllers/arithmeticController.js]: you have break the function. please revert the change
